src/page/Signup.tsx

```
@@ -1,63 +1,195 @@
-import { useState } from 'react';
+import React, { useState } from 'react';
 import {
   Map,
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)